test/playwright.spec.js

```diff
@@ -181,6 +181,9 @@ module.exports.describe = ({testRunner, product, playwrightPath}) => {
         testRunner.loadTests(require('./features/permissions.spec.js'), testOptions);
       }
 
+      if (WEBKIT) {
+        testRunner.loadTests(require('./webkit/provisional.spec.js'), testOptions);
```

Comment:
I like that we extracted these into their own file. Is there any reason we cant run them for other browsers? While we don't do provisional stuff in the protocol for chromium, it still often had similar bugs related to process swapping.

Reply:
Chromium does not resolve most protocol promises during navigation, and that's unlikely to change. Basically, we are not testing API here (because we cannot make it work everywhere), but specifically webkit implementation.

Reply:
fair.

Reply:
So we should eventually upstream these into protocol tests in webkit itself.